src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,19 @@
+.forecast-details {
+  margin: 1rem auto;
```

Comment:
Mainly the units here should be looked at as before. Also, the xx-large font size doesnt make a lot of sense in  context, given the title of the element is bigger... Theres a lot of ways to do this, but I would defer to using rem for fonts most of the time